internal/api/core/core.go

```diff
@@ -29,10 +29,11 @@ import (
 	"github.com/perses/perses/internal/api/schemas"
 	"github.com/perses/perses/pkg/model/api/config"
 	"github.com/perses/perses/ui"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/sirupsen/logrus"
 )
 
-func New(conf config.Config, banner string) (*app.Runner, dependency.PersistenceManager, error) {
+func New(conf config.Config, banner string, metricNamespace string, registry *prometheus.Registry) (*app.Runner, dependency.PersistenceManager, error) {
```

Comment:
Rather than passing `metricNamespace` here, would it make sense to just use `utils.MetricNamespace` directly? `view` already uses `utils.MetricNamespace` so it would be nice to be consistent

Reply:
ah yeah totally I forgot to change that ! Thanks :)